geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -134,4 +171,21 @@ app.MeasureController = function($scope, ngeoDecorateInteraction) {
 };
 
 
+/**
+ * @param {ol.Coordinate} coordinates
+ * @return {angular.$q.Promise}
+ * @private
+ */
+app.MeasureController.prototype.getElevation_ = function(coordinates) {
+  var lonlat =
```

Comment:
I mean: is it really a longitude and a latitude that you store in that variable?